tests/parsers/agilent_gen5/testdata/absorbance/010307_114129_BNCH654563_stdcurve_singleplate01.json

```diff
@@ -40,21 +35,20 @@
                             }
                         }
                     ],
+                    "measurement time": "2022-10-10T21:00:04+00:00",
+                    "plate well count": {
+                        "value": 288,
```

Comment:
Plate well count is still not correct - this value should not change from the previous implementation. 

Looks like you need to count the number of well positions, not the number of measurements?

Reply:
@joshua-benchling, agree with Nate here. We should count number of positions rather than measurements. Likely expected values for the field should be 96 or 384 in most cases.